keyboards/dz60/keymaps/ottodokto/rules.mk

```diff
@@ -0,0 +1,59 @@
+# MCU name
```

Comment:
The entire contents of this file can be minimized down to:

```make
VELOCIKEY_ENABLE = yes
```

The settings here override the settings in `keyboards/dz60/rules.mk`, but the Velocikey rule is the only thing you actually need; everything else will be inherited from keyboard level.